packages/toolpad-app/src/toolpad/AppEditor/PageEditor/JsExpressionEditor.tsx

```diff
@@ -44,9 +44,17 @@ export function JsExpressionEditor({
       .map((key) => `declare const ${key}: RootObject[${JSON.stringify(key)}];`)
       .join('\n');
 
+    // TODO: cleanup after Toolpad v1
+    const queryDeprecation = `
+      /** @deprecated use parameters */
+      declare const query: Query;
+    `;
+
     const content = `
       ${type.join('\n')}
 
+      // TODO: cleanup after Toolpad v1
+      ${globalScope.query && globalScope.parameters ? queryDeprecation : ''}
```

Comment:
@Janpot I know this isn't the most elegant way, but I'm not sure if we should invest more if we plan to improve/rework this part entirely?